src/richie/apps/courses/templates/courses/cms/blogpost_detail.html

```diff
@@ -13,6 +13,9 @@
         <meta property="og:image:width" content="{{ thumb.width }}" />
         <meta property="og:image:height" content="{{ thumb.height }}" />
     {% endblockplugin %}
+    {% if not current_page|is_empty_placeholder:"excerpt" %}
+        <meta property="og:description" content="{% filter striptags %}{% show_placeholder 'excerpt' current_page %}{% endfilter %}" />
```

Comment:
I think it should be trimed and trunked the same as the meta description.

Reply:
Yes. Similar approach. Trunked to 200 characters.